pygamelib/engine.py

```diff
@@ -3119,3 +3119,63 @@ def display_at(
             eol = self.terminal.clear_eol
         with self.terminal.location(column, row):
             print(text, eol, end=end, file=file, flush=flush)
+    
+    def display_sprite_at(
+        self,
+        sprite,
+        row=0,
+        column=0,
+        file=sys.stdout,
+        flush=False,
+    ):
+        """
+        Displays sprite at a given position.
+
+        :param sprite: The sprite object to display.
+        :type sprite: pygamelib.gfx.core.Sprite
+        :param row: The row position in the terminal window.
+        :type row: int
+        :param column: The column position in the terminal window.
+        :type column: int
+        :param file:
+        :type file: stream
+        :param flush: print() parameter to flush the stream after printing
+        :type flush: bool
+
+        Example::
+        
+            screen.display_sprite_at(panda_sprite,
+                                     int(screen.height/2),
+                                     int(screen.width/2)
+                                     )
+        """
+        for r in range(0, sprite.size[1]):
+            for c in range(0, sprite.size[0]):
+                self.display_at(
+                    sprite._sprixels[r][c], row + r, column + c, file=file, flush=flush
+                )
+    
+    def display_sprite(self, sprite, file=sys.stdout, flush=False):
+        """
+        Displays sprite at the current cursor position.
+        
+        :param sprite: The sprite object to display.
+        :type sprite: pygamelib.gfx.core.Sprite
+        :param file:
+        :type file: stream
+        :param flush: print() parameter to flush the stream after printing
+        :type flush:
+
+        Examples::
+
+            screen.display_sprite(panda_sprite)
+        """
+        for r in range(0, sprite.size[1]):
+            for c in range(0, sprite.size[0]):
+                if sprite._sprixels[r][c] == core.Sprixel():
+                    print(" ", end="")
```

Comment:
This needs to be documented, could you add something like that in the documentation: if a sprixel is empty it's going to be replaced by a space character.
Actually, what do you think of adding an extra parameter named `filler` or something like that.
It would have a default value of `Sprixel(' ')` and could be changed to achieve different results (like changing colors for example).
This could also be offered in display_at (even though we don't need it). It would be for purely aesthetic considerations.
In any case, if we make a decision to replace something in the sprite, we need to document it.

Reply:
Thanks for the feedback. I updated the documentation and added `filler` parameter.